statement_test.go

```diff
@@ -89,6 +89,11 @@ func TestBuildStatement(t *testing.T) {
 			input: "DROP TABLE t1",
 			want:  &DdlStatement{Ddl: "DROP TABLE t1"},
 		},
+		{
+			desc:  "DROP TABLE statement",
```

Comment:
Is this test case outdated?

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/49/commits/eda6bf845b4ad79ed72f8d7eab38b7e7b17d9e15